src/resources/postcss/views/day/_event.pcss

```diff
@@ -9,12 +9,14 @@
 		flex-direction: column;
 		margin-bottom: var(--spacer-5);
 		margin-top: var(--spacer-7);
+		padding-left: calc(var(--grid-width-1-of-4) - (var(--grid-gutter-small-half) / 2));
```

Comment:
There needs to be spacing in the calc for the `-` sign, otherwise building the pcss files fails. A general rule to follow is to add spaces around operators. I fixed this for you.

Reply:
Thanks! Didn't know that.